src/plans/user-interface/upgrade-sidebar-menu-integration.php

```diff
@@ -103,17 +104,23 @@ public function register_hooks() {
 	 * @return array<string, array<string, array<static|string>>> The pages.
 	 */
 	public function add_page( $pages ) {
+		// Don't show the Upgrade button if Yoast SEO WooCommerce addon is active.
+		$addon_manager = new WPSEO_Addon_Manager();
```

Comment:
This class is part of the DI container, so let's add the `WPSEO_Addon_Manager` as a property of the class (much like how we do eg. in the `Bust_Subscription_Cache_Route` class)